src/validation.cpp

```diff
@@ -828,7 +804,6 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
             return state.Invalid(TxValidationResult::TX_MEMPOOL_POLICY, "insufficient fee",
                     strprintf("rejecting replacement %s, not enough additional fees to relay; %s < %s",
                         hash.ToString(),
-                        FormatMoney(nDeltaFees),
```

Comment:
This line removal must be unintentional (three placeholders, but only two arguments)? Quite a pity that the compiler doesn't warn us in this case. At least the tinyformat library seems to be nice enough to notice us at runtime :)
```
$ ./test/functional/feature_rbf.py
.....
.....
2021-08-14T18:32:49.823000Z TestFramework (INFO): Running test replacement relay fee...
2021-08-14T18:32:49.834000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):                                                                                                                                                             File "/home/honey/bitcoin_prrev/test/functional/test_framework/util.py", line 132, in try_rpc
    fun(*args, **kwds)
  File "/home/honey/bitcoin_prrev/test/functional/test_framework/coverage.py", line 49, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/home/honey/bitcoin_prrev/test/functional/test_framework/authproxy.py", line 146, in __call__
    raise JSONRPCException(response['error'], status)
test_framework.authproxy.JSONRPCException: tinyformat: Too many conversion specifiers in format string (-1)
```

Reply:
woah good catch O.o

Reply:
Fixed